src/app/components/challenge/challengesubmit/challengesubmit.component.ts

```diff
@@ -22,6 +22,7 @@ inputErrorMessage = '';
 urlboxchecked = false;
 fileboxchecked = false;
 validFileUrl = false;
+onSubmissionOption: boolean;
```

Comment:
Please change this name to `isSubmissionUsingUrl`.

Reply:
but it reflect option file and url

Reply:
i have done the change